from src.adder import addme

assert(addme(10, 20) == 30)
print("Successful call. If it was failure I wouldnt have got printed")
